docs/usage/transports/azure-sb.md

```diff
@@ -43,6 +43,8 @@ The functions [host.json](https://docs.microsoft.com/en-us/azure/azure-functions
 }
 ```
 
+>Please be aware that when hosted in Azure Functions using Azure Service Bus as the transport, MassTransint won't create the topology for you, so this needs to be tackled by you either manually or at deployment time.
```

Comment:
Typo on MassTransit.
This might be a good point to also cover the question of **how** to achieve this at deployment time - perhaps reference the use of the DeployTopologyOnly  configuration flag

Reply:
Oh damn,

yeah I was actually thinking to add a link to a deployment page, where we can document the best solution to deploy MassTransit applications

Reply:
@phatboyg What you think here Chris? Worth adding a note on DeployTopologyOnly? I couldn't see any references to this flag anywhere - is this the best place to introduce?

Reply:
I think a deployment guidance page under the learn/getting help section would be enough. Could put that consumers should be deployed before producers, etc. and how to use the deploy flag.

Reply:
Sounds fair - probably goes separate to this one but can be referenced from here

Reply:
Is this one good to go like this then?